tempodb/encoding/common/config.go

```diff
@@ -65,5 +65,9 @@ func ValidateConfig(b *BlockConfig) error {
 		return fmt.Errorf("positive value required for bloom-filter shard size")
 	}
 
+	if b.Version == "vParquet" {
```

Comment:
can be replaced with this constant:

https://github.com/grafana/tempo/blob/main/tempodb/encoding/vparquet/encoding.go#L14

Reply:
ah, i'm good with this then 👍